proj/src/main/webapp/chat.js

```diff
@@ -428,23 +428,29 @@ function initRef(dbRefObject) {
 
 function pushChatMessage() {
     const messageInput = document.getElementById('message-input');
-
-    var message = {
+    // prevent blank messages
+    if (messageInput.value.trim().length != 0){
+        var message = {
         content : messageInput.value,
```

Comment:
I'm not sure if it is what you want, but maybe you should include the `.trim()` here also when you're storing it. It means you can't have messages prefixed or suffixed with white space which can either be good or bad. Just for you to think about.

Reply:
this maybe very niche but i think trimming spaces would mess up those ASCII pic/art people occasionally send in chat